code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -669,42 +759,40 @@
 
 /obj/item/clothing/head/hooded/winterhood/security
 	desc = "A red, armour-padded winter hood. Definitely not bulletproof, especially not the part where your face goes."
-	icon_state = "winterhood_security"
+	icon_state = "winterhood_sec"
 	armor = list(MELEE = 25, BULLET = 15, LASER = 30, ENERGY = 40, BOMB = 25, BIO = 0, RAD = 0, FIRE = 0, ACID = 45)
 
-/obj/item/clothing/suit/hooded/wintercoat/security/hos
-	name = "head of security's winter coat"
-	desc = "A red, armour-padded winter coat, lovingly woven with a Kevlar interleave and reinforced with semi-ablative polymers and a silver azide fill material. The zipper tab looks like a tiny replica of Beepsky."
-	icon_state = "coathos"
-	inhand_icon_state = "coathos"
-	armor = list(MELEE = 35, BULLET = 25, LASER = 40, ENERGY = 50, BOMB = 35, BIO = 0, RAD = 0, FIRE = 0, ACID = 55)
-	hoodtype = /obj/item/clothing/head/hooded/winterhood/security/hos
-
-/obj/item/clothing/head/hooded/winterhood/security/hos
-	desc = "A red, armour-padded winter hood, lovingly woven with a Kevlar interleave. Definitely not bulletproof, especially not the part where your face goes."
-	icon_state = "winterhood_hos"
+/obj/item/clothing/suit/hooded/wintercoat/security/det
+	name = "\improper detective's winter coat"
+	desc = "A random dectective decided to throw away the typical trenchcoat stariotype and make himself a wintercoat, this was the result."
+	icon_state = "wintercoat_det"
+	inhand_icon_state = "wintercoat_det"
+	armor = list("melee" = 35, "bullet" = 30, "laser" = 30,"energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 50, "acid" = 50)
```

Comment:
this should copy the stat's of the detective's trenchcoat, not his vest, or else it'll outclass both of them (the former in armor values, and the latter in area coverage)